content/sprint-review-2110.md

```diff
@@ -0,0 +1,42 @@
+Title: Wazo Platform 21.10 Released
+Date: 2021-07-30T17:25:00
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2110
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 21.10 release.
+
+## New Features in This Release
+
+* CDR: Improve CDR to add information about the source_internal name
+* Call permission: It's now possible to use the same name on different tenants
+
+## Technical Features
+
+* Asterisk: Asterisk has been upgraded to 18.5.1. See the [Asterisk release announcement](https://www.asterisk.org/asterisk-news/asterisk-13-38-3-16-19-1-17-9-4-18-5-1-and-16-8-cert10-now-available-security/)
+* The implementation of the directory plugin for Google has been updated to use the People API instead of the deprecated Contact API. See the upgrade note for more details about the changes that have to be done.
```

Comment:
```suggestion
* **Google**: The implementation of the directory plugin for Google has been updated to use the People API instead of the deprecated Contact API. See the upgrade note for more details about the changes that have to be done.
```